app/models/review.rb

```diff
@@ -19,4 +19,24 @@ def visible?
       return false
     end
   end
+
+  def get_project
+    if project
+      return project.title
+    end
+  end
+
+  def get_project_owner
+    if project and project.project_owner
+      return project.project_owner.user.name
+    end
+  end
+
+  def self.get_random_review(user)
+    review = Review.offset(rand(Review.count)).first
+    while review.user == user do
+      review = Review.offset(rand(Review.count)).first
```

Comment:
If `review = Review.offset(rand(Review.count)).first` is repeated, then isn't the `while` unnecessary?